src/core/algorithms/statistics/data_stats.cpp

```diff
@@ -606,6 +594,272 @@ Statistic DataStats::GetAvgNumberOfChars(size_t index) const {
     return Statistic(res, &double_type, false);
 }
 
+template <class Pred>
+Statistic DataStats::GetMinOf(size_t index, Pred pred) const {
```

Comment:
```suggestion
Statistic DataStats::GetStringMinOf(size_t index, Pred pred) const {
```
Same for other `Get...Of` because this is only applicable for this data type.